packages/govuk-frontend/src/govuk/components/header/header.mjs

```diff
@@ -53,22 +53,30 @@ export class Header extends GOVUKFrontendComponent {
     }
 
     this.$module = $module
-    this.$menuButton = $module.querySelector('.govuk-js-header-toggle')
-    this.$menu =
-      this.$menuButton &&
-      $module.querySelector(
-        `#${this.$menuButton.getAttribute('aria-controls')}`
-      )
-
-    if (
-      !(
-        this.$menuButton instanceof HTMLElement ||
-        this.$menu instanceof HTMLElement
-      )
-    ) {
+    const $menuButton = $module.querySelector('.govuk-js-header-toggle')
+
+    // Header don't necessarily have a navigation, in which case
```

Comment:
Do we need to watch our wording/naming here or in the component?

1. **`<nav>`** for `.govuk-header__navigation`
2. **`<ul>`** for `govuk-header__navigation-list`

Confusingly in the JavaScript `this.$menu` is the **`<ul>`** but has `id="navigation"

```suggestion
    // Headers don't necessarily have a navigation, in which case
```

Reply:
Whoops, sorry for the typo.

Regarding the wording, I was referring to what's toggled by the `navigation` option of the Nunjucks component, which would be the `<nav>`, in the comment. Agreed it's a bit unfortunate that the `id` is `navigation` for the `__navigation-list` element. Would be a breaking change for people not using the Nunjucks macro, so worth its own discussion if we want to change it (not sure the current situation hurts much, but not fussed if the team prefers updating).

Reply:
> Do we need to watch our wording/naming here **or in the component?**

Cheers for fixing that. Probably more the last bit?

I'll link another comment to this thread